packages/relay/src/api/index.ts

```diff
@@ -10,10 +10,75 @@ import {
   SignedMessagePayload,
   SignTypedDataPayload,
   SignMessagePayload,
+  CreateRelayerRequest,
+  RelayerListResponse,
+  UpdateRelayerPoliciesRequest,
+  UpdateRelayerRequest,
 } from '../relayer';
 
 export const RelayerApiUrl = () => process.env.DEFENDER_RELAY_API_URL || 'https://api.defender.openzeppelin.com/';
 
+export class RelayClient extends BaseApiClient {
+  protected getPoolId(): string {
+    return process.env.DEFENDER_RELAY_CLIENT_POOL_ID || 'us-west-2_94f3puJWv';
+  }
+
+  protected getPoolClientId(): string {
+    return process.env.DEFENDER_RELAY_CLIENT_POOL_CLIENT_ID || '40e58hbc7pktmnp9i26hh5nsav';
+  }
+
+  protected getApiUrl(): string {
+    return process.env.DEFENDER_RELAY_CLIENT_API_URL || 'https://defender-api.openzeppelin.com/';
+  }
+
+  public async get(relayerId: string): Promise<RelayerModel> {
```

Comment:
In `defender` you were using `RelayerGetResponse`. I know these are copied from the `repo`, but I'd also prefer consistency with the naming, what do you think?


```suggestion
  public async get(relayerId: string): Promise<RelayerGetResponse> {
```

Reply:
This does match the convention better - updated

But I think removing `RelayerModel` would break TypeScript users who had imported that type from our repo, so I maintained it as a deprecated reference. Do we have any systems around maintaining deprecations?

Reply:
You got a point there. I think it does not matter that much if we break the compatibility for the users since they have to upgrade first. It's is the typical broken-when-updated thing, so it's safe to assume that if they're upgrading or installing the first time (without a `.lock` file) it'll warn them.

I mean, it's not at risk to be deployed and broken. Anyway, I've only seen deprecation in the backend but in this case it doesn't apply.